backend-app/.env.example

```diff
@@ -1,8 +1,13 @@
+NODE_ENV = "development"
 API_VERSION = "v1"
-MONGO_URI = "mongodb+srv://username:password@cluster0.cmlpjag.mongodb.net/?retryWrites=true&w=majority"
+MONGO_URI = "mongodb+srv://swo:zrcoCze9UKFTWzYx@nfs315.tayahgd.mongodb.net/S-W-O"
+PORT = 5000
 ADMIN_EMAIL = "admin@swf.com"
 ADMIN_PASSWORD = "password123418746"
 JWT_SECRET = "JWT-Seceret-Key"
```

Comment:
Use this to generate a real JWT_SECRET : 
```shel
node -e "console.log(require('crypto').randomBytes(128).toString('hex'))"
```

Reply:
sure